resources/views/auth/login.blade.php

```diff
@@ -7,6 +7,8 @@
             <div class="">
                 <div class="card-header bg-transparent p-3 text-center font-weight-bold h3">{{ __('Login') }}</div>
 
+		<a href="{{ route('oidc.signin') }}">Login with Keycloak</a>
```

Comment:
I assume will want check something like

`OIDC_ENABLED=true`

here so it's now showing to instances that don't use it

Reply:
I've added a `config('oidc.enabled')` check in the two login files